change/@ni-nimble-components-8d042e30-91cd-421a-99af-5d4bb68493b0.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Just test / story changes, no source code changes, so no release needed

```suggestion
  "type": "none",
```